### DISCUSSION TITLE
Git LFS has been disabled on your personal account

### DISCUSSION THREAD

**Discussion by Code-Fashioned** [community/community#27478] [Other Feature Feedback, Questions, & Ideas]
"Git LFS has been disabled on your personal account UmbralScythe because you’ve exceeded your data plan by at least 150%. Please purchase additional data packs to cover your bandwidth and storage usage:

 Bandwidth: 0.0 GB / 1 GB (0%)

  Storage: 1.89 GB / 1 GB (189%)"
  
This is the email the sent me this moring :c I did a Git Bush to transfer my assets to the lfs server 'cause of a problem (now solved) is there a way I can Fix this?

**Comment by airtower-luna** [comment]
I assume you want to delete the LFS files that you don't need any more? What the [documentation has to say on the topic](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository) is unfortunately not that great: Either delete and recreate the repository, or [ask GitHub Support for help](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket).
